.circleci/config.yml

```diff
@@ -88,7 +88,8 @@ jobs:
       - run:
           name: Test
           command: |
-            detox test --configuration ios.sim.release --cleanup
+            detox clean-framework-cache && detox build-framework-cache
+            detox test --configuration ios.sim.release --cleanup --loglevel verbose
```

Comment:
Why verbose?
Doesn't it makes everything slower on CI?

Reply:
it's easier to debug why tests failed. I removed it.